src/server/accessors/ModifyUpdater.ts

```diff
@@ -69,6 +69,6 @@ export class ModifyUpdater implements IModifyUpdater {
             throw new Error('Invalid type assigned to the room.');
         }
 
-        return this.bridges.getRoomBridge().update(result, this.appId);
+        return this.bridges.getRoomBridge().update(result, builder.getMembersToBeAddedUsernames(), this.appId);
```

Comment:
Doing this will actually add members to the room if they don't exist already? And will it check permissions? If so, how will that be checked? This gets into the complicated world...

Reply:
Yes, it will add if executer has permission, it will not check permissions since I didn't see any other part checking permissions I'm assuming that the app has permission for everything right now.